source/style-guide/style/email-addresses.txt

```diff
@@ -4,14 +4,10 @@
 Email Addresses
 ===============
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 If you need to show an example email address, use **example.com** or
 **example.org** as the domain. The Internet Assigned Numbers Authority
 (IANA) reserves these domains for use in examples.
 
-Additionally, format example email addresses as bold. For example,
-**yourName@example.com**.
-
-If you need to show an actual email address, use the convention in your
-authoring environment to make the email address live.
+If you need to show an actual email address, use a `standalone hyperlink
+<https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#standalone-hyperlinks>`_
```

Comment:
```suggestion
If you need to show an actual email address, use a :rst:`standalone hyperlink
</restructuredtext.html#standalone-hyperlinks>`
```

Reply:
Didn't realize we had an :rst: role. Noted.